mp/src/game/shared/momentum/mom_gamemovement.cpp

```diff
@@ -2528,8 +2552,17 @@ int CMomentumGameMovement::TryPlayerMove(Vector *pFirstDest, trace_t *pFirstTrac
         //  for contact
         // Add it if it's not already in the list!!!
         MoveHelper()->AddToTouched(pm, mv->m_vecVelocity);
-        if (!CloseEnough(pm.plane.normal[2], 0.0f)) // Filter out straight walls
-            m_pPlayer->SetLastCollision(pm);
+
+        if (player->GetGroundEntity() == nullptr)
+        {
+            SurfInt::Type type = SurfInt::TYPE_WALL;
+            if (pm.plane.normal.z > 0.0f)
+                type = SurfInt::TYPE_FLOOR;
+            else if (pm.plane.normal.z < 0.0f)
+                type = SurfInt::TYPE_CEILING;
```

Comment:
Use braces, the only time braces don't have to be used is if it's a return/single logic statement:
```
if ( bEarlyCheck )
    return (whatever);
```
or
```
if ( bSomeCheck )
    incrementSomeValue();

. . . ( no else )
```